packages/checkout/shipping/shipping-rates-control/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as ShippingRatesControl } from './shipping-rates-control';
+export { default as Packages } from './packages';
```

Comment:
Don't think we need to export package.

Reply:
Sounds good, you can remove the export here then :)